L/LibGit2/build_tarballs.jl

```diff
@@ -62,7 +62,7 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     Dependency("MbedTLS_jll"; compat="~2.28.0"),
-    Dependency("LibSSH2_jll"; compat="1.10.1"),
+    Dependency("LibSSH2_jll"; compat="^1.10.1"),
```

Comment:
The rebuilt libssh2 is the newest version, 1.11.0, and I'm try to get the build to use that without dropping compatibility with earlier libssh2. Julia is currently using 1.10.2 but I figured I would update to 1.11.0, unless it's simpler to create separate libssh2 builders for 1.10 and 1.11.